sunkit_pyvista/plotter.py

```diff
@@ -72,6 +74,25 @@ def set_camera_coordinate(self, coord):
         pos = tuple(camera_position[0])
         self.plotter.camera.position = pos
 
+    def set_view_angle(self, angle: u.deg = 30):
+        """
+        Sets the view angle of the camera by the specified value
+        in degrees.
```

Comment:
```suggestion
```

Reply:
Same issue with self.camera.view_angle being in degrees

Reply:
You don't need this because we have used a quantity input - this will allow input in other units as long as they are convertible to degrees.

Reply:
Yes, I misunderstood this, sorry